src/reynir_correct/errtokenizer.py

```diff
@@ -2069,10 +2109,11 @@ def is_valid_suggestion(
             assert isinstance(corrected, tuple)
             corrected_display = " ".join(corrected)
             for ix, corrected_word in enumerate(corrected):
-                if ix == 0:
+                rtok = token
+                if ix == 0 and "S001" not in ignore_rules:
                     rtok = replace_word(1, token, corrected_word, corrected_display)
                     at_sentence_start = False
-                else:
+                elif "S001" not in ignore_rules:
```

Comment:
After thinking on it, I isolated the ignore_rules check as close to the error annotation as possible, because we want this if loop to pick this up as usual, just skipping the error annotation.